Engines/Wine/Settings/Font smoothing/script.js

```diff
@@ -0,0 +1,82 @@
+include("engines.wine.engine.object");
+include("engines.wine.plugins.regedit");
+include("engines.wine.plugins.font_smoothing");
+
+/**
+ * setting to set the Fonts Smoothing
+*/
+var settingImplementation = {
+    _options: [tr("Default"), tr("RGB"), tr("BGR"), tr("Gray Scale")],
+
+    _registryValues: [],
+
+    getText: function () {
+        return tr("Fonts Smoothing");
+    },
+    getOptions: function () {
+        return this._options;
+    },
+    getCurrentOption: function (container) {
+        var FontSmoothing = new Wine()
+            .prefix(container)
+            .regedit()
+            .fetchValue(["HKEY_CURRENT_USER", "Control Panel", "Desktop", "FontSmoothing"]);
+
+        var FontSmoothingType = new Wine()
+            .prefix(container)
+            .regedit()
+            .fetchValue(["HKEY_CURRENT_USER", "Control Panel", "Desktop", "FontSmoothingType"]);
+
+        var FontSmoothingOrientation = new Wine()
+            .prefix(container)
+            .regedit()
+            .fetchValue(["HKEY_CURRENT_USER", "Control Panel", "Desktop", "FontSmoothingOrientation"]);
+
+        var index;
+
+        if (FontSmoothing == 1) {
+            index = 0;
+        }
+        else {
+            if (FontSmoothingType == 2) {
+                if (FontSmoothingOrientation == 1) {
+                    index = 1;
+                }
+                else {
+                    index = 2;
+                }
+            }
+            else if (FontSmoothingType == 1) {
+                index = 3;
+            }
+        }
+
+        return this._options[index];
+    },
+    setOption: function (container, optionIndex) {
+        var regeditFileContent;
+        if (0 == optionIndex) {
```

Comment:
`==` or `===`?

Reply:
What do you prefer ? As I understand, `===` just do not any type conversion, right ?

Reply:
In general I would say: If you are not completely sure, prefer `===` to avoid surprises.